approvals/src/main/java/no/sikt/nva/approvals/domain/Identifier.java

```diff
@@ -0,0 +1,5 @@
+package no.sikt.nva.approvals.domain;
+
+public record Identifier(String type, String value) {
+
+}
```

Comment:
Is type meant for the "type of identifier" (CTIS/REK etc)? If so, could be a problem if we also add the field "type" for Jackson serialization later (like we often do).